src/Whoops/Exception/Frame.php

```diff
@@ -119,7 +119,11 @@ public function getFileContents()
                 return null;
             }
 
-            $this->fileContentsCache = file_get_contents($filePath);
+            try {
+                $this->fileContentsCache = file_get_contents($filePath);
```

Comment:
Why not returning the result here? Otherwise `return null` in the catch block makes no sense because it'll look weird as a replacement.

Reply:
You are right.
It also could be just an empty catch block, because `fileContentsCache` is `null` anyways.